Solutions/Recorded Future/Package/createUiDefinition.json

```diff
@@ -64,7 +64,7 @@
             "name": "workbooks-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs workbooks to help you gain insights into the telemetry collected in Microsoft Sentinel. After installing the solution, start using the workbook in Manage solution view."
+              "text": "This Microsoft Sentinel Solution installs workbooks. Workbooks provide a flexible canvas for data monitoring, analysis, and the creation of rich visual reports within the Azure portal. They allow you to tap into one or many data sources from Microsoft Sentinel and combine them into unified interactive experiences."

```

Comment:
@RecordedFutureOskbo, Can you please check and confirm if the change for workbook description needed? Thanks.

Reply:
This text is generated via the \Tools\Create-Azure-Sentinel-Solution\V2\createSolutionV2.ps1 script and I have no control of the input. It is not needed for the solution. But it's nice to provide the generated files without us changing them.

Reply:
@RecordedFutureOskbo, ok thanks for the update.